src/components/PokemonForm.js

```diff
@@ -1,35 +1,72 @@
-import React from "react";
-import { Form } from "semantic-ui-react";
+import React, { useState } from "react"
+import { Form } from "semantic-ui-react"
+
+function PokemonForm({ onSubmitPokemon }) {
+  const [name, setName] = useState("")
+  const [hp, setHp] = useState("")
+  const [frontUrl, setFrontUrl] = useState("")
+  const [backUrl, setBackUrl] = useState("")
```

Comment:
I'd really love to see you go back and try to implement a single state, maybe called`formData`, to manage the form data, instead of all separate states. If you have time, this would be a good refactor.

Reply:
went back added single state